dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -590,7 +590,23 @@ const backgroundHeader = (format: ArticleFormat): string => {
 const backgroundStandfirst = (format: ArticleFormat): string => {
 	switch (format.design) {
 		case ArticleDesign.LiveBlog:
-			return pillarPalette[format.theme][300];
+			switch (format.theme) {
+				case ArticlePillar.News:
+					return news[200];
+				case ArticlePillar.Culture:
+					return culture[200];
+				case ArticlePillar.Sport:
+					return sport[200];
```

Comment:
Think this is `sport[100]`? Here's `editorialPalette` for this: https://github.com/guardian/dotcom-rendering/blob/72e90fe8963b4e8d8c87ad30678c61b536de288b/common-rendering/src/editorialPalette/background.ts#L82-L87

Reply:
Ditto @HarryFischer

Reply:
Actually, I don't need to check with Harry, I can see in the designs that you're right - great spot!